lib/pages/chat_list/chat_list_view_style.dart

```diff
@@ -11,6 +11,7 @@ class ChatListViewStyle {
       const Size.fromHeight(120);
 
   // Between 0 and 1, scale on actions length
-  static const double slidableExtentRatio = 0.25;
+  static const double slidableExtentRatio = 0.50;
```

Comment:
why we use the ratio, how about use fixed size? IMO. right now it is too big.

Reply:
Sizes are determined like that on slidable lib, extent ratio is the total length of the drawer. I'll check for better solutions